Annotation/polar_golden_set.json

```diff
@@ -855,13 +940,13 @@
             {
                 "predicate": [
                     {
-                        "str": "irrelevant",
+                        "str": "to be totally irrelevant",
                         "lemma": "",
                         "POS": ""
                     }
                 ],
                 "type": "polar",
-                "clause": "",
+                "clause": "whether he is able or not to shoot good pictures",

```

Comment:
Again, here we have a clause as a subject, so it's not an embedded clause (I think)